### PR TITLE
Mark the partition as bootable

### PR DESCRIPTION
**Pull request by atomicbitty**
Some BIOSes (eg. my laptop's) require at least one partition to be marked as bootable, this patch can save one from frustration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,8 @@ and install qloader2, one can do:
 dd if=/dev/zero bs=1M count=0 seek=64 of=test.img
 parted -s test.img mklabel msdos
 parted -s test.img mkpart primary 1 100%
+parted -s loader.img set 1 boot on
```
Comment (VisualDevelopment) [inline README.md:56]: The image is called test.img, not loader.img

**Review by VisualDevelopment — CHANGES_REQUESTED** [review]

**Comment by atomicbitty** [comment]
Sorry for the mistake

**Review by VisualDevelopment — APPROVED** [review]
:+1: Looks good to me!

**Comment by mintsuki** [comment]
Thank you @atomicbitty and @VisualDevelopment and WOW I cannot believe how terrible BIOSes can be. BIOSes should not care about the partition table, but alas they do, and it can be very annoying.

I suggest it be specified with a comment that setting an active partition is a workarond for potentially buggy BIOSes rather than something that qloader2 actually cares about.

Other than that, LGTM

File: README.md [hunk 2]
```diff
@@ -53,6 +53,8 @@ and install qloader2, one can do:
 dd if=/dev/zero bs=1M count=0 seek=64 of=test.img
 parted -s test.img mklabel msdos
 parted -s test.img mkpart primary 1 100%
+parted -s test.img set 1 boot on
```
Comment (VisualDevelopment) [inline README.md:56]:
```suggestion
parted -s test.img set 1 boot on # Workaround for buggy BIOSes
```

**Comment by VisualDevelopment** [comment]
Also, @mintsuki wait till you see the BIOSes that need some weird jmp at the start to mark the boot sector as valid :^)

**Comment by atomicbitty** [comment]
@mintsuki some bios are way worse, some set the boot disk number to 0x00 with hard disks (it actually happened with a laptop which I tested - laptops have horrible bioses) and it results to disk read error. I may send another pull request for that.

**Comment by VisualDevelopment** [comment]
> @mintsuki some bios are way worse, some set the boot disk number to 0x00 with hard disks (it actually happened with a laptop which I tested - laptops have horrible bioses) and it results to disk read error. I may send another pull request for that.

I actually noticed a workaround for this in GRUB code

**Comment by atomicbitty** [comment]
yeah, i've done this exact same workaround in my bootloader.

**Comment by mintsuki** [comment]
> @mintsuki some bios are way worse, some set the boot disk number to 0x00 with hard disks (it actually happened with a laptop which I tested - laptops have horrible bioses) and it results to disk read error. I may send another pull request for that.

Sure feel free to send a PR for that too. That is incredibly weird though, AFAIK drive 0 is supposed to be the A: floppy disk drive. Could it be that the BIOS just loads up the bootsector thinking it's a floppy's?

**Comment by atomicbitty** [comment]
> Could it be that the BIOS just loads up the bootsector thinking it's a floppy's?
Probably not, because I was explicitly saying to the bios to emulate the usb drive as HDD. The problem with most proprietary BIOSes is that they are only tested with Windows and maaaaaybe linux or BSD. And these OSes are already protected against dumb BIOSes, so the bugs are (almost) never noticed.

> BIOSes should not care about the partition table, but alas they do, and it can be very annoying.
All proprietary BIOSes I tested either required a partition table (HDDs) or a BPB (FDDs) in order to boot.

**Comment by VisualDevelopment** [comment]
I hate hardware vendors lol

**Pull request merged**